src/main/java/com/cloudant/client/api/views/ViewRequest.java

```diff
@@ -46,6 +46,38 @@
     ViewResponse<K, V> getResponse() throws IOException;
 
     /**
+     * Performs the request for the given pagination token and returns the response.
```

Comment:
"Performs view request for the page represented by a pagination token obtained from a previous request"?